helm-charts/0.25.4/charts/keystone/values.yaml

```diff
@@ -15,59 +15,60 @@ replicaCount: 1
 
 livenessPort: 8081
 
-# See https://vsecm.com/configuration for more information
+# -- See https://vsecm.com/configuration for more information
 # about these environment variables.
-
 environments:
-  - name: SPIFFE_ENDPOINT_SOCKET
-    value: "unix:///spire-agent-socket/agent.sock"
+  # -- The log level. 0: Logs are off (only audit events will be logged)
+  # 7: TRACE level logging (maximum verbosity).
   - name: VSECM_LOG_LEVEL
     value: "7"
-  - name: VSECM_WORKLOAD_SPIFFEID_PREFIX
-    value: "spiffe://vsecm.com/workload/"
-  - name: VSECM_SAFE_TLS_PORT
-    value: ":8443"
 
+# -- See https://vsecm.com/configuration for more information
+# about these environment variables.
 initEnvironments:
+  # -- The SPIFFE endpoint socket. This is used to communicate with the SPIRE 
+  # agent. If you change this, you will need to change the associated 
+  # volumeMount in the Deployment.yaml too.
   - name: SPIFFE_ENDPOINT_SOCKET
     value: "unix:///spire-agent-socket/agent.sock"
+  # -- The log level. 0: Logs are off (only audit events will be logged)
+  # 7: TRACE level logging (maximum verbosity).
   - name: VSECM_LOG_LEVEL
     value: "7"
-  - name: VSECM_WORKLOAD_SPIFFEID_PREFIX
-    value: "spiffe://vsecm.com/workload/"
+  # -- The interval (in milliseconds) that the VSecM Init Container will poll
```

Comment:
These are mostly massaging the documentation to align with `helm-docs` format with as minimal changes as possible.

We can always improve this later down the line.